src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx

```diff
@@ -281,6 +282,24 @@ All parameters are case-insensitive.</value>
     (runspace-terminating) error, such as a throw or -ErrorAction Stop, occurs
     or when execution is interrupted with Ctrl-C.
 
+-CommandWithArgs | -cwa
+
+    Executes a PowerShell command with arguments.  Unlike `-Command`, this
+    parameter populates the `$args built-in variable which can be used by the
```

Comment:
```suggestion
    parameter populates the `$args` built-in variable, which can be used by the
```

Reply:
I should probably indicate in the built-in help that this is `[Experimental]` so will make that change.  Since this feature is opt-in (you have to use the new parameter) and pwsh wouldn't read `powershell.config.json` to see if the feature is enabled, it's a catch-22 so best is to have the ExperimentalFeature to indicate the design may change or the feature could be removed.